content/04-postgresql/14-short-guides/04-exporting-schemas.mdx

```diff
@@ -0,0 +1,123 @@
+```
+
+Depending on your PostgreSQL configuration, you may need to also include some of the following options:
+
+* `--username=` / `-U`: The database username that you want to authenticate with
+* `--password` / `-W`: Force `pg_dump` to prompt for a password to authenticate
+* `--host=` / `-h`: The hostname or IP address of where PostgreSQL is located
+* `--port=` / `-p`: The port number where PostgreSQL is listening
+
+The option responsible for telling `pg_dump` to only include the database structures, not the data itself is `--schema-only`:
+
+* `--schema-only`: `-s`: Export only the object definitions, not the data itself.
+
+Additionally, the first non-option argument (represented here by the word "DATABASE") indicates the exact database to export.
+
+Using this information, you could export the schema of a database called `SALES` using a limited user called `sales_reporter` with a command like this:
+
+```command
+pg_dump --username=sales_reporter --password --schema-only SALES > sales_database_schema.sql
+```
+
+## Modifying the export behavior
+
+The basic usage discussed above will output every structure related to the database in question.  We can modify this behavior with a number of additional options and in some cases, with the related `pg_dumpall` command.
+
+### Targeting more than one database
+
+The `pg_dump` command is designed to export information related to a single database.  To target more than one database at a time, you can instead substitute the `pg_dumpall` command, which follows a similar syntax.
+
+To dump all of the database schemas in a PostgreSQL cluster, type:
+
+```command
+pg_dumpall --schema-only > all_schemas.sql
+```
+
+If you want to limit the databases that are dumped, you can optionally include the `--exclude-database=` option.  This can be used multiple times to target individual databases:
+
+```command
+pg_dumpall --schema-only --exclude-database=FIRST --exclude-database=SECOND > almost_all_schemas.sql
+```
+
+It can also use wildcard-style matching to catch multiple databases with a single pattern:
+
+```command
+pg_dumpall --schema-only --exclude-database='SALES_*'  > all_schemas_except_sales.sql
+```
+
+### Exporting only certain structures
+
+You can also reduce the structures exported by naming specific tables to export:
+
+* `--table=` / `-t`: Dump only the tables that match the given pattern.  Can be provided multiple times.
+* `--exclude-table=` / `-T`: Exclude the tables that match the given pattern.  Can be provided multiple times.
+
+For example, if three of the tables in your `SALES` database are called `EMPLOYEE`, `STORE`, and `INVENTORY`, all defined in the default `public` schema, you can export only those structures by typing:
+
+> **Note:** You can find out more about how PostgreSQL interprets different types of quotation marks in our guide on [how to use single and double quotes in PostgreSQL](https://www.prisma.io/dataguide/postgresql/short-guides/quoting-rules).
```

Comment:
The note feels better after the code example when viewing the preview of the page. Up to you :)